Gems/Atom/RPI/Code/Source/RPI.Public/RenderPipeline.cpp

```diff
@@ -352,8 +368,8 @@ namespace AZ
 
                 pipelineViews.m_views[0]->SetPassesByDrawList(nullptr);
                 m_persistentViewsByViewTag.erase(pipelineViews.m_views[0].get());
-                m_pipelineViewsByTag.erase(viewTag);
```

Comment:
Is erasing the `viewTag` not necessary anymore?

Reply:
During `SetPersistentView` the engine only adds the view to existing entries in `m_pipelineViewsByTag`, and doesn't create new entries, which means once we de-register a view, we could not register another view for that view-tag until `m_pipelineViewsByTag`  was (re)-build.
This did happen at the beginning of most frames while we tested this, which is why we didn't notice this earlier.